### PR TITLE
[Keyboard] Added a new handwired 2x3,2x4,2x5 keyboard called the Stream_cheap

### PR DESCRIPTION
**Pull request by hartk1213**
stream cheap is a diy version of the El Gato Stream deck minus the LCD keys
but you can always get relegendable keycaps to change the icon if you want

## Description
stream cheap is a diy version of the El Gato Stream deck minus the LCD keys
but you can always get relegendable keycaps to change the icon if you want

## Types of Changes
added new files didnt change any old stuff, my new keyboard can be a 2x3, 2x4, or a 2x5 macro pad the case is 3d printed and is available [here](https://www.thingiverse.com/thing:4497991)

- [ ] Core
- [ ] Bugfix
- [ ] New feature
- [ ] Enhancement/optimization
- [x] Keyboard (addition or update)
- [x] Keymap/layout/userspace (addition or update)
- [x] Documentation

## Issues Fixed or Closed by This PR

* 

## Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the [**CONTRIBUTING** document](https://docs.qmk.fm/#/contributing).
- [ ] I have added tests to cover my changes.
- [x] I have tested the changes and verified that they work and don't break anything (as well as I can manage).

### CODE REVIEW

File: keyboards/handwired/stream_cheap/2x3/2x3.c [hunk 1]
```diff
@@ -0,0 +1,7 @@
+#include "2x3.h"
+
+
+void matrix_init_kb(void)
+{
+    matrix_init_user();
+}
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x3/2x3.c:7]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x3/config.h [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Tap dancing params */
+#define TAPPING_TERM 250
+
+
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x3/config.h:33]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x3/rules.mk [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+COMMAND_ENABLE = no     # Commands for debug and configuration
+CONSOLE_ENABLE= no	# Console for debug
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+MOUSEKEY_ENABLE = yes	# Mouse keys
+NKRO_ENABLE = yes	# USB Nkey Rollover -
+
+RGBLIGHT_ENABLE = no
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+UNICODE_ENABLE = yes
+TAP_DANCE_ENABLE = no
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x3/rules.mk:27]:
```suggestion
# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/handwired/stream_cheap/2x4/2x4.c [hunk 4]
```diff
@@ -0,0 +1,7 @@
+#include "2x4.h"
+
+
+void matrix_init_kb(void)
+{
+    matrix_init_user();
+}
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x4/2x4.c:7]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x4/config.h [hunk 5]
```diff
@@ -0,0 +1,31 @@
...
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Tap dancing params */
+#define TAPPING_TERM 250
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x4/config.h:31]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x4/info.json [hunk 6]
```diff
@@ -0,0 +1,22 @@
...
+			"key_count":8,
+            "layout": [
+                {"label":"K00", "x":0, "y":0},
+                {"label":"K01", "x":1, "y":0},
+                {"label":"K02", "x":2, "y":0},
+                {"label":"K02", "x":3, "y":0},
+                {"label":"K03", "x":0, "y":1},
+                {"label":"K04", "x":1, "y":1},
+                {"label":"K05", "x":2, "y":1},
+                {"label":"K05", "x":3, "y":1}
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x4/info.json:18]:
```suggestion
                {"label":"K03", "x":3, "y":0},
                {"label":"K04", "x":0, "y":1},
                {"label":"K05", "x":1, "y":1},
                {"label":"K06", "x":2, "y":1},
                {"label":"K07", "x":3, "y":1}
```

File: keyboards/handwired/stream_cheap/2x4/rules.mk [hunk 7]
```diff
@@ -0,0 +1,27 @@
...
+COMMAND_ENABLE = no     # Commands for debug and configuration
+CONSOLE_ENABLE= no	# Console for debug
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+MOUSEKEY_ENABLE = yes	# Mouse keys
+NKRO_ENABLE = yes	# USB Nkey Rollover -
+
+RGBLIGHT_ENABLE = no
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+UNICODE_ENABLE = yes
+TAP_DANCE_ENABLE = no
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x4/rules.mk:27]:
```suggestion
# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```

File: keyboards/handwired/stream_cheap/2x5/2x5.c [hunk 8]
```diff
@@ -0,0 +1,7 @@
+#include "2x5.h"
+
+
+void matrix_init_kb(void)
+{
+    matrix_init_user();
+}
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x5/2x5.c:7]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x5/config.h [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+#define DEBOUNCE 5
+
+/* Mechanical locking support. Use KC_LCAP, KC_LNUM or KC_LSCR instead in keymap */
+#define LOCKING_SUPPORT_ENABLE
+
+/* Locking resynchronize hack */
+#define LOCKING_RESYNC_ENABLE
+
+/* Tap dancing params */
+#define TAPPING_TERM 250
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x5/config.h:30]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x5/rules.mk [hunk 10]
```diff
@@ -0,0 +1,27 @@
...
+COMMAND_ENABLE = no     # Commands for debug and configuration
+CONSOLE_ENABLE= no	# Console for debug
+EXTRAKEY_ENABLE = yes	# Audio control and System control
+MOUSEKEY_ENABLE = yes	# Mouse keys
+NKRO_ENABLE = yes	# USB Nkey Rollover -
+
+RGBLIGHT_ENABLE = no
+SLEEP_LED_ENABLE = no   # Breathing sleep LED during USB suspend
+UNICODE_ENABLE = yes
+TAP_DANCE_ENABLE = no
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x5/rules.mk:27]:
```suggestion
```suggestion
# Bootloader selection
BOOTLOADER = caterina

# Build Options
#   change yes to no to disable
#
BOOTMAGIC_ENABLE = lite     # Virtual DIP switch configuration
MOUSEKEY_ENABLE = yes       # Mouse keys
EXTRAKEY_ENABLE = yes       # Audio control and System control
CONSOLE_ENABLE = no         # Console for debug
COMMAND_ENABLE = no         # Commands for debug and configuration
# Do not enable SLEEP_LED_ENABLE. it uses the same timer as BACKLIGHT_ENABLE
SLEEP_LED_ENABLE = no       # Breathing sleep LED during USB suspend
# if this doesn't work, see here: https://github.com/tmk/tmk_keyboard/wiki/FAQ#nkro-doesnt-work
NKRO_ENABLE = yes           # USB Nkey Rollover
BACKLIGHT_ENABLE = no       # Enable keyboard backlight functionality
RGBLIGHT_ENABLE = no        # Enable keyboard RGB underglow
BLUETOOTH_ENABLE = no       # Enable Bluetooth
AUDIO_ENABLE = no           # Audio output
```
```

File: keyboards/handwired/stream_cheap/2x5/info.json [hunk 11]
```diff
@@ -0,0 +1,24 @@
...
+                {"label":"K00", "x":0, "y":0},
+                {"label":"K01", "x":1, "y":0},
+                {"label":"K02", "x":2, "y":0},
+                {"label":"K02", "x":3, "y":0},
+                {"label":"K02", "x":4, "y":0},
+                {"label":"K03", "x":0, "y":1},
+                {"label":"K04", "x":1, "y":1},
+                {"label":"K05", "x":2, "y":1},
+                {"label":"K05", "x":3, "y":1},
+                {"label":"K05", "x":4, "y":1}
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/2x5/info.json:20]:
```suggestion
                {"label":"K00", "x":0, "y":0},
                {"label":"K01", "x":1, "y":0},
                {"label":"K02", "x":2, "y":0},
                {"label":"K03", "x":3, "y":0},
                {"label":"K04", "x":4, "y":0},
                {"label":"K10", "x":0, "y":1},
                {"label":"K11", "x":1, "y":1},
                {"label":"K12", "x":2, "y":1},
                {"label":"K13", "x":3, "y":1},
                {"label":"K14", "x":4, "y":1}
```

File: keyboards/handwired/stream_cheap/config.h [hunk 12]
```diff
@@ -0,0 +1,25 @@
+#pragma once
```
Comment (zvecr) [inline keyboards/handwired/stream_cheap/config.h:1]: This file should be removed

File: keyboards/handwired/stream_cheap/2x3/config.h [hunk 13]
```diff
@@ -0,0 +1,33 @@
...
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1213
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Kyle Hart
+#define PRODUCT         stream_cheap
+#define DESCRIPTION     stream cheap
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x3/config.h:11]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x3/info.json [hunk 14]
```diff
@@ -0,0 +1,20 @@
+{
+    "keyboard_name": "Stream Cheap 2x3",
+    "url": "https://www.thingiverse.com/thing:4497991",
+    "maintainer": "qmk",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_2x3": {
+			"key_count":6,
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x3/info.json:9]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x4/config.h [hunk 15]
```diff
@@ -0,0 +1,31 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1213
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Kyle Hart
+#define PRODUCT         stream_cheap
+#define DESCRIPTION     stream cheap
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x4/config.h:11]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x4/info.json [hunk 16]
```diff
@@ -0,0 +1,22 @@
+{
+    "keyboard_name": "Stream Cheap 2x4",
+    "url": "https://www.thingiverse.com/thing:4497991",
+    "maintainer": "qmk",
+    "width": 4,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_2x4": {
+			"key_count":8,
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x4/info.json:9]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x4/keymaps/default/keymap.c [hunk 17]
```diff
@@ -0,0 +1,43 @@
...
+            }
+            break;
+        case VS_UNCOMMENT:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCTL("ku"));
+            }
+            break;
+        case TST_STRING:
+            if (record->event.pressed) {
+                SEND_STRING("12345"SS_TAP(X_ENTER));
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x4/keymaps/default/keymap.c:30]:
```suggestion
                SEND_STRING("12345\n");
```

File: keyboards/handwired/stream_cheap/2x5/config.h [hunk 18]
```diff
@@ -0,0 +1,30 @@
...
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1213
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Kyle Hart
+#define PRODUCT         stream_cheap
+#define DESCRIPTION     stream cheap
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x5/config.h:11]:
```suggestion
```

File: keyboards/handwired/stream_cheap/2x5/info.json [hunk 19]
```diff
@@ -0,0 +1,24 @@
+{
+    "keyboard_name": "Stream Cheap 2x5",
+    "url": "https://www.thingiverse.com/thing:4497991",
+    "maintainer": "qmk",
+    "width": 5,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_2x5": {
+			"key_count":10,
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x5/info.json:9]:
```suggestion
```

File: keyboards/handwired/stream_cheap/config.h (hunk 12 above)
Comment (hartk1213) [reply keyboards/handwired/stream_cheap/config.h]: file has been removed

File: keyboards/handwired/stream_cheap/2x5/keymaps/default/keymap.c [hunk 20]
```diff
@@ -0,0 +1,57 @@
...
+            }
+            break;
+        case VS_UNCOMMENT:
+            if (record->event.pressed) {
+                SEND_STRING(SS_LCTL("ku"));
+            }
+            break;
+        case TST_STRING:
+            if (record->event.pressed) {
+                SEND_STRING("12345"SS_TAP(X_ENTER));
```
Comment (fauxpark) [inline keyboards/handwired/stream_cheap/2x5/keymaps/default/keymap.c:25]:
```suggestion
                SEND_STRING("12345\n");
```

Comment (hartk1213) [reply keyboards/handwired/stream_cheap/2x5/keymaps/default/keymap.c:25]: Just curious what is different between /n and ss_tap(x_enter) ?

I'll make the change I was just curious

Comment (fauxpark) [reply keyboards/handwired/stream_cheap/2x5/keymaps/default/keymap.c]: Functionally, there is no difference, the enter key will still be sent. However `SS_TAP(X_ENTER)` compiles to `0x01 0x01 0x28` ([The first two bytes represent a tap event](https://github.com/qmk/qmk_firmware/blob/master/quantum/send_string_keycodes.h#L394)), whereas simply adding `\n` to the string results in just `0x0A`, the ASCII line feed character, which is already mapped to `KC_ENTER` in the [sendstring lookup table](https://github.com/qmk/qmk_firmware/blob/master/quantum/quantum.c#L391). So, it just saves a couple bytes and a bit of extra computation.

File: keyboards/handwired/stream_cheap/2x3/2x3.h [hunk 21]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_2x3( \
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x3/2x3.h:5]:
```suggestion
#define LAYOUT_ortho_2x3( \
```

File: keyboards/handwired/stream_cheap/2x3/config.h [hunk 22]
```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1213
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Kyle Hart
+#define PRODUCT         stream_cheap
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x3/config.h:10]:
```suggestion
#define PRODUCT         Stream Cheap 2x3
```

File: keyboards/handwired/stream_cheap/2x3/info.json [hunk 23]
```diff
@@ -0,0 +1,19 @@
+{
+    "keyboard_name": "Stream Cheap 2x3",
+    "url": "https://www.thingiverse.com/thing:4497991",
+    "maintainer": "qmk",
+    "width": 3,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_2x3": {
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x3/info.json:8]:
```suggestion
        "LAYOUT_ortho_2x3": {
```

File: keyboards/handwired/stream_cheap/2x3/keymaps/default/keymap.c [hunk 24]
```diff
@@ -0,0 +1,16 @@
...
+  NORMAL_LAYER = 0,
+   SECOND_LAYER
+};
+
+
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [NORMAL_LAYER] = LAYOUT_2x3(KC_1, KC_2, KC_3,KC_4, KC_5, TO(1)),
+	[SECOND_LAYER]  = LAYOUT_2x3(KC_A, KC_B, KC_C,KC_D, KC_E, TO(0))
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x3/keymaps/default/keymap.c:13]:
```suggestion
    [NORMAL_LAYER] = LAYOUT_ortho_2x3(
        KC_1, KC_2, KC_3,
        KC_4, KC_5, TO(1)
    ),
    [SECOND_LAYER] = LAYOUT_ortho_2x3(
        KC_A, KC_B, KC_C,
        KC_D, KC_E, TO(0)
    )
```

File: keyboards/handwired/stream_cheap/2x4/2x4.h [hunk 25]
```diff
@@ -0,0 +1,12 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_2x4(\
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x4/2x4.h:5]:
```suggestion
#define LAYOUT_ortho_2x4(\
```

File: keyboards/handwired/stream_cheap/2x4/config.h [hunk 26]
```diff
@@ -0,0 +1,27 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1213
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x4/config.h:7]:
```suggestion
#define PRODUCT_ID      0x1214
```

File: keyboards/handwired/stream_cheap/2x4/config.h [hunk 27]
```diff
@@ -0,0 +1,27 @@
...
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Kyle Hart
+#define PRODUCT         stream_cheap
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x4/config.h:10]:
```suggestion
#define PRODUCT         Stream Cheap 2x4
```

File: keyboards/handwired/stream_cheap/2x4/info.json [hunk 28]
```diff
@@ -0,0 +1,21 @@
+{
+    "keyboard_name": "Stream Cheap 2x4",
+    "url": "https://www.thingiverse.com/thing:4497991",
+    "maintainer": "qmk",
+    "width": 4,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_2x4": {
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x4/info.json:8]:
```suggestion
        "LAYOUT_ortho_2x4": {
```

File: keyboards/handwired/stream_cheap/2x4/keymaps/default/keymap.c [hunk 29]
```diff
@@ -0,0 +1,43 @@
...
+
+    }
+    return true;
+}
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [NORMAL_LAYER] = LAYOUT_2x4(VS_COMMENT, VS_UNCOMMENT, TST_STRING, KC_4,
+							KC_5, KC_6, KC_7, TO(1)),
+	[SECOND_LAYER] = LAYOUT_2x4(KC_A, KC_B, KC_C, KC_D,
+							KC_E, KC_F, KC_G, TO(0))
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x4/keymaps/default/keymap.c:42]:
```suggestion
    [NORMAL_LAYER] = LAYOUT_ortho_2x4(
        VS_COMMENT, VS_UNCOMMENT, TST_STRING, KC_4,
        KC_5,       KC_6,         KC_7,       TO(1)
    ),
    [SECOND_LAYER] = LAYOUT_ortho_2x4(
        KC_A, KC_B, KC_C, KC_D,
        KC_E, KC_F, KC_G, TO(0)
    )
```

File: keyboards/handwired/stream_cheap/2x5/2x5.h [hunk 30]
```diff
@@ -0,0 +1,11 @@
+#pragma once
+
+#include "quantum.h"
+
+#define LAYOUT_2x5( \
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x5/2x5.h:5]:
```suggestion
#define LAYOUT_ortho_2x5( \
```

File: keyboards/handwired/stream_cheap/2x5/config.h [hunk 31]
```diff
@@ -0,0 +1,26 @@
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x1213
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x5/config.h:7]:
```suggestion
#define PRODUCT_ID      0x1215
```

File: keyboards/handwired/stream_cheap/2x5/config.h [hunk 32]
```diff
@@ -0,0 +1,26 @@
...
+#define DEVICE_VER      0x0001
+#define MANUFACTURER    Kyle Hart
+#define PRODUCT         stream_cheap
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x5/config.h:10]:
```suggestion
#define PRODUCT         Stream Cheap 2x5
```

File: keyboards/handwired/stream_cheap/2x5/info.json [hunk 33]
```diff
@@ -0,0 +1,23 @@
+{
+    "keyboard_name": "Stream Cheap 2x5",
+    "url": "https://www.thingiverse.com/thing:4497991",
+    "maintainer": "qmk",
+    "width": 5,
+    "height": 2,
+    "layouts": {
+        "LAYOUT_2x5": {
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x5/info.json:8]:
```suggestion
        "LAYOUT_ortho_2x5": {
```

File: keyboards/handwired/stream_cheap/2x5/keymaps/default/keymap.c [hunk 34]
```diff
@@ -0,0 +1,56 @@
...
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+
+    [NORMAL_LAYER] = LAYOUT_2x5(VS_COMMENT, VS_UNCOMMENT, LOCK, TST_STRING, KC_F5,
+							                  KC_F9, KC_F10, KC_F11, KC_F12, TO(1)),
+	[SECOND_LAYER] = LAYOUT_2x5(KC_MUTE, KC_VOLD, KC_VOLU, KC_MPLY, KC_MSTP,
+							                CUT, COPY, PASTE, FIND,TO(0))
+              // ,
+	// [THIRD_LAYER] = LAYOUT_2x5(KC_Q, KC_W, KC_E, KC_R, KC_T,
+	// 						KC_Y, KC_U, KC_L, KC_P,TO(0))
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/2x5/keymaps/default/keymap.c:54]:
```suggestion
    [NORMAL_LAYER] = LAYOUT_ortho_2x5(
        VS_COMMENT, VS_UNCOMMENT, LOCK, TST_STRING, KC_F5,
        KC_F9, KC_F10, KC_F11, KC_F12, TO(1)
    ),
    [SECOND_LAYER] = LAYOUT_ortho_2x5(
        KC_MUTE, KC_VOLD, KC_VOLU, KC_MPLY, KC_MSTP,
        CUT,     COPY,    PASTE,   FIND,    TO(0)
    )
    // ,
    // [THIRD_LAYER] = LAYOUT_ortho_2x5(
    //      KC_Q, KC_W, KC_E, KC_R, KC_T,
    //      KC_Y, KC_U, KC_L, KC_P, TO(0)
    // )
```

File: keyboards/handwired/stream_cheap/readme.md [hunk 35]
```diff
@@ -0,0 +1,23 @@
...
+
+Keyboard Maintainer: [qmk](https://github.com/qmk)  
+
+Hardware:
+  * 3d printed switch plate
+  * 10 key switches
+  * 3d printed case
+  * Handwired in direct pins mode (i.e. all switches share a common gnd and are wired to thier own pin)
+  * Uses a Pro Micro
+  * Includes a reset switch accessible by a hole on the bottom
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/readme.md:15]:
```suggestion
An ortholinear macropad, available in 2x3, 2x4, and 2x5 sizes.

Hardware:
  * 3d printed switch plate
  * 6-10 key switches
  * 3d printed case
  * Handwired in Direct Pins mode (i.e. all switches share a common GND and are wired to their own pin)
  * Uses a Pro Micro
  * Includes a reset switch accessible by a hole on the bottom

* Keyboard Maintainer: [hartk1213](https://github.com/hartk1213)
* Hardware Supported: Pro Micro
* Hardware Availability: [Thingiverse](https://www.thingiverse.com/thing:4497991)
```

File: keyboards/handwired/stream_cheap/readme.md [hunk 36]
```diff
@@ -0,0 +1,23 @@
+# stream_cheap
```
Comment (noroadsleft) [inline keyboards/handwired/stream_cheap/readme.md:1]:
```suggestion
# Stream Cheap
```

**Comment by noroadsleft** [comment]
Thanks!

**Pull request merged**